app/controllers/player_inputs_controller.rb

```diff
@@ -11,19 +11,23 @@ def create
     @round.save
     @current_user = current_or_guest_user
 
-    @player_input = PlayerInput.create!(
+    @player_input = PlayerInput.new(
       instance_id: @instance.id,
       player: Player.find_by(user_id: @current_user.id),
       input_type: "string",
       input_value: params[:input_value],
       round_id: @round.id
     )
 
-    # send to channel
-    RoundChannel.broadcast_to(
-      @round,
-      render_to_string(partial: "player_input", locals: { player_input: @player_input })
-    )
+    if @player_input.save
+      # send to channel
+      RoundChannel.broadcast_to(
+        @round,
+        render_to_string(partial: "player_input", locals: { player_input: @player_input })
+      )
+    else
+      flash[:alert] = "You already replied" unless @player_input.save
```

Comment:
We already have `if @player_input.save` on line 22 so I don't think we need `unless @player_input.save` on this line.

I tried to played with this in my local but my fingers aint fast enough to trigger the flash alert.

Reply:
oh my god you're absolutely right! will fix it right away sorry about that!

Reply:
Solved :D